api/nakadi-event-bus-api.yaml

```diff
@@ -1617,6 +1618,34 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /topics/{name}:
+    get:
```

Comment:
This is not very RESTful. When I do GET /topics/myTopic I expect to get some topic object but instead I get EventType (for which there is a separate resource)
So the endpoint should be more like "get event-type by topic"

I think the endpoint
`GET /event-types?topic=9bc91968-101b-435b-b697-33d47f8c69c4` will fit more